packages/expo-image/src/Image.tsx

```diff
@@ -1,21 +1,26 @@
 import React from 'react';
 import {
   AccessibilityProps,
+  ImageResizeMode,
   ImageSourcePropType,
   ImageStyle,
   NativeSyntheticEvent,
   StyleProp,
+  StyleSheet,
 } from 'react-native';
 
 import ExpoImage from './ExpoImage';
 import { ImageErrorEventData, ImageLoadEventData, ImageLoadProgressEventData } from './Image.types';
 
+const DEFAULT_RESIZE_MODE = 'cover';
+
 export interface ImageProps extends AccessibilityProps {
   // On one hand we want to pass resolved source to native module.
   // On the other hand, react-native-web doesn't expose a resolveAssetSource
   // function, so we can't use it there. So we pass the unresolved source
   // to "native components" and they decide whether to resolve the value
   // or not.
+  resizeMode?: ImageResizeMode;
```

Comment:
Could you please put it not immediately after the comment describing the `source` prop (so either before the comment or after the `source` prop) so the comment stays colocated with the prop it talks about?

Reply:
Ah I hadn't noticed that. I thought the props were in alphabetical order that's why I put it there.
I've moved it to the bottom for now.
Would you reckon having the props in alphabetical order is a good thing?